bundles/org.openhab.binding.emotiva/src/main/java/org/openhab/binding/emotiva/internal/EmotivaProcessorHandler.java

```diff
@@ -264,46 +218,48 @@ private void scheduleConnectRetry(long waitMinutes) {
     }
 
     /**
-     * Starts a polling job for connection to th device, adds the
+     * Starts a polling job for connection to the device, adds the
      * {@link EmotivaBindingConstants#DEFAULT_KEEP_ALIVE_IN_MILLISECONDS} as a time buffer for checking, to avoid
      * flapping state or minor network issues.
      */
     private void startPollingKeepAlive() {
         final ScheduledFuture<?> localRefreshJob = this.pollingJob;
         if (localRefreshJob == null || localRefreshJob.isCancelled()) {
-            logger.debug("Start polling");
-
-            int delay = stateMap.get(EmotivaSubscriptionTags.keepAlive.name()) != null
-                    && stateMap.get(EmotivaSubscriptionTags.keepAlive.name()) instanceof Number keepAlive
-                            ? keepAlive.intValue()
-                            : config.keepAlive;
-            pollingJob = scheduler.scheduleWithFixedDelay(this::checkKeepAliveTimestamp,
-                    delay + DEFAULT_KEEP_ALIVE_IN_MILLISECONDS, delay + DEFAULT_KEEP_ALIVE_IN_MILLISECONDS,
+
+            int keepAliveTime = state.getChannel(EmotivaSubscriptionTags.keepAlive)
+                    .filter(channel -> channel instanceof Number)
+                    .map(keepAlive -> (Number) ((Number) keepAlive).intValue())
+                    .orElse(config.keepAlive).intValue();
```

Comment:
Spotless suggests this, did you run it?
```suggestion
                    .map(keepAlive -> (Number) ((Number) keepAlive).intValue())
                    .orElse(config.keepAlive).intValue();
```